examples/inputs/gch.yaml

```diff
@@ -112,7 +112,8 @@ flow_field:
 
   ###
   # The level of turbulence intensity level in the wind.
-  turbulence_intensity: 0.06
+  turbulence_intensities:
+  - 0.06
```

Comment:
This is the only change so far to the floris input files from v3 to v4 (that `turbulence_intensity`, a scalar, changes to `turbulence_intensities`, a list). When running the new code with an v3 input file, the generic error gets raised:

`AttributeError: The initialization for FlowField was given extraneous inputs: ['turbulence_intensity']`

Is that OK, or do we need either a converter to convert v3 models to v4 (similar to what we now have for v3-style turbine models) or a more explicit explanation of the change in the error raised?

Reply:
Both sound good to me, we could add a specific error for when turbulence_intensity is provided, and maybe (with a seperate issue) prepare a converter?